feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries.context.spec.ts

```diff
@@ -44,7 +50,15 @@ class MockProductImportInfoService {
   getResults = createSpy().and.returnValue(of(mockProductImportInfo));
 }
 
-describe('NewSavedCartOrderEntriesContext', () => {
+const mockFailingProductImportInfo = {
+  productCode: 'testProductCode',
+  statusCode: ProductImportStatus.UNKNOWN_IDENTIFIER,
```

Comment:
Can we add a product case where the status code is UNKNOWN_ERROR?

Reply:
Hey, pushed a commit regarding this, let me know if you'd like something changed!